rs/sns/integration_tests/src/root.rs

```diff
@@ -44,6 +44,7 @@ fn test_get_status() {
                 latest_ledger_archive_poll_timestamp_seconds: None,
                 index_canister_id: Some(PrincipalId::new_user_test_id(45)),
                 testflight: false,
+                updated_framework_canisters_memory_limit: Some(true),
```

Comment:
Please check if this test is also needed for `updated_framework_canisters_memory_limit: None,`

Reply:
It does not seem to be